components/builder-web/app/package/package/package.component.ts

```diff
@@ -38,4 +43,19 @@ export class PackageComponent implements OnDestroy {
             name: this.name
         };
     }
+
+    onRouteActivate(routedComponent) {
```

Comment:
Is this checking to see if the currently routed component is in this array, and if so, enable the sidebar?

Reply:
Yep. Specifically it's being used to apply a class that shows the sidebar (as opposed to re-rendering it), but yes, that's the gist.